uktt.gemspec

```diff
@@ -17,7 +17,7 @@ Gem::Specification.new do |spec|
   end
 
   spec.require_paths = %w[lib]
-  spec.required_ruby_version = '>= 3.0.2'
+  spec.required_ruby_version = ">= #{File.read('.ruby-version').chomp}"
```

Comment:
is not better just to put 3.0.3 here ? (since it is hardcoded in circle ci config.yml)